typescript/packages/well-completions-plot/demo/example-data/well-completions.json

```diff
@@ -0,0 +1,2390 @@
+{
```

Comment:
This file already exists in /example-data. Can we use the top-level file instead of duplicating it ?

Reply:
This file is not used for the `WellCompletionsPlot`, so I'll delete it. However the `dash-components/WellCompletions`-package is a rename and move of the original/refactored `WellCompletions`-package, which has the same files in its folder `WellCompletions/example-data`. Is it preferred to have a top-level folder for example-data, or keep the example-data close  to the owner?